index.html

```diff
@@ -18,14 +30,34 @@ <h1 class="center">Awesome Books</h1>
       </table>
     <br>
     <hr><br>
-    <div class="form-container">
-      <form id="book-form">
-      <h2 class="center">Add a new book</h2><br>
-      <input type="text" id="title" name="title" required placeholder="Title"><br><br>
-      <input type="text" id="author" name="author" required placeholder="Author"><br><br>
-      <button type="submit" class="submit center">Add</button>
-    </form>
-  </div>
+    </section>
+    <section class="add-new-section">
+      <div class="form-container">
+        <form id="book-form">
+        <h2 class="center">Add a new book</h2><br>
+        <input type="text" id="title" name="title" required placeholder="Title"><br><br>
+        <input type="text" id="author" name="author" required placeholder="Author"><br><br>
+        <button type="submit" class="submit center">Add</button>
```

Comment:
- [ ] Everything looks good but there is an error when I tried to add a book, I got this error. I have attached the screenshot below.

![Screenshot from 2023-02-25 03-12-52](https://user-images.githubusercontent.com/26767383/221299368-97939aac-acfe-47d7-8db3-90ef96cdb1ea.png)